docs/adr/0006-forms.md

```diff
@@ -59,33 +59,33 @@ These libraries can exist side by side in the code base. New forms can be built
 
 ### Incident form
 
-- src/signals/incident/components/IncidentForm/index.js
+- [IncidentForm/index.js](../../src/signals/incident/components/IncidentForm/index.js)
 
 Two form components specifically using the dependency. These can easily be updated when using the new lib.
 
-- src/signals/incident/components/form/EmphasisCheckboxInput/index.js
-- src/signals/incident/components/form/Header/index.js
+- [form/Header/index.js](../../src/signals/incident/components/form/Header/index.js)
```

Comment:
This link doesn't work

Reply:
There were two of theme next to each other. Removed one. The link works for me however. Does it work for you now?

Reply:
In VSCode doesn't link correct

Reply:
Ah, it does on Github.